src/AI.cpp

```diff
@@ -28,21 +28,31 @@ void AI::connect() {
     // connect to server as AI
     if (!libClientHandler.network.connect(address, port)) {
         spdlog::critical("could not connect to server");
-        std::exit(2);
+        exit(2);
     }
     spdlog::info("connected to server");
     if (!libClientHandler.network.sendHello(name, spy::network::RoleEnum::AI)) {
         spdlog::critical("could not send Hello message");
-        std::exit(2);
+        exit(2);
     }
     spdlog::info("send Hello message");
+}
 
-    // TODO make sure that matchConfig is available
+void AI::requestConfigs() {
+    using namespace spy::network::messages;
+    if (!libClientHandler.network.sendRequestMetaInformation(
+            {MetaInformationKey::CONFIGURATION_MATCH_CONFIG, MetaInformationKey::CONFIGURATION_SCENARIO,
+             MetaInformationKey::CONFIGURATION_CHARACTER_INFORMATION})) {
+        spdlog::critical("could not send RequestMetaInformation message for configs");
+        exit(2);
```

Comment:
Ich würde hier
```suggestion
        throw std::runtime_error{"could not send RequestMetaInformation message for configs"};
```
bevorzugen, auch an den stellen oben.

Das erlaubt es, den fehler an späterer stelle zu behandeln. Hat außerdem den vorteil, dass lokale objekte destroyed werden, da der scope verlassen wird (das hat man mit exit nicht)

Reply:
Ok, habe das überall gemacht wo nach dem logging ein exit Aufruf war